src/quicktype-json5-input/index.ts

```diff
@@ -0,0 +1,26 @@
+import { parse as JSON5parse } from "json5";
+import { CompressedJSONFromString, JSONInput, TargetLanguage, languageNamed, defined } from "../quicktype-core";
+
+export class CompressedJSON5FromString extends CompressedJSONFromString {
+    parseSync(input: string): number {
+        const json = JSON5parse(input);
+        this.process(json);
+        return this.finish();
+    }
+}
+
+export class JSON5Input<T> extends JSONInput<T> {
+    readonly kind: string = "json5";
+}
+
+export function json5InputForTargetLanguage(
```

Comment:
Where is json5 support more appropriate? A new pkg `quicktype-json5-input` or `quicktype-core`?